### PR TITLE
Data normalization 2

### PR DESCRIPTION
**Pull request by alexandernacho**

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -344,6 +344,85 @@ By setting this to `latest`, you will get the most recent version. The returned
...
+  "timestamp": 1540474596175,
+  "trade_id": "68024786",
+  "price": "6559.84000000",
+  "amount": "0.00646600",
+  "taker_side_sell": true
+}
+```
+
+### Buy/Sell, Maker/Taker sides
+For trades, the way exchanges report on the type of trade varies greatly. Some exchanges only report whether a trade was a buy or sell, while others specify the side the maker or taker was on. With the following mapping, we try to rule out any ambiguous interpretation of the `taker_side_sell` field in our trade endpoints.
```
Comment (Legogris) [inline source/index.html.md]: Some exchanges only report whether a trade was a buy or sell, without specifying which side of the trade this is actually referring to.

File: source/index.html.md [hunk 2]
```diff
@@ -344,6 +344,85 @@ By setting this to `latest`, you will get the most recent version. The returned
...
+  "trade_id": "68024786",
+  "price": "6559.84000000",
+  "amount": "0.00646600",
+  "taker_side_sell": true
+}
+```
+
+### Buy/Sell, Maker/Taker sides
+For trades, the way exchanges report on the type of trade varies greatly. Some exchanges only report whether a trade was a buy or sell, while others specify the side the maker or taker was on. With the following mapping, we try to rule out any ambiguous interpretation of the `taker_side_sell` field in our trade endpoints. 
+
```
Comment (Legogris) [inline source/index.html.md]: Add clarification that the following is in reference to what exchanges are reporting via their APIs

File: source/index.html.md [hunk 3]
```diff
@@ -344,6 +344,85 @@ By setting this to `latest`, you will get the most recent version. The returned
...
+#### Maker aggressor
```
Comment (Legogris) [inline source/index.html.md]: Maker side ("Agressor" == taker: https://www.investopedia.com/terms/a/aggressor.asp)

File: source/index.html.md [hunk 4]
```diff
@@ -344,6 +344,85 @@ By setting this to `latest`, you will get the most recent version. The returned
...
+| ---                         | ---                                        | ---                                                   | ---
+| Binance<sup>1</sup>         | `"m": true, false`                         | `"m": true`                                           | 
+| Coinbase Pro<sup>1 2</sup>    | `"side": "buy" or "sell"`                  | `"side": "sell"`                                      |
+| Gemini<sup>2</sup>          | `"type": "buy" or "sell"`                  | `"type": "sell"`                                      | 
+| ZB.com<sup>2</sup>          | `"type": "buy" or "sell"`                  | `"type": "sell"`                                      | 
+<sup>1</sup> Explicitely stated in docs
+
+<sup>2</sup> Confirmed by exchange
+
+#### Taker agressor
```
Comment (Legogris) [inline source/index.html.md]: Taker side

File: source/index.html.md (hunk 2 above)
Comment (Legogris) [reply source/index.html.md]: s/refererrence/reference

Comment (Legogris) [reply source/index.html.md]: s/exchagnes/exchanges

Comment (Legogris) [reply source/index.html.md]: I'd remove "We contacted the exchanges for which the documentation did not specify details on the side or for which the meaning was not fully clear. ". Or reword.

File: source/index.html.md [hunk 5]
```diff
@@ -344,6 +344,82 @@ By setting this to `latest`, you will get the most recent version. The returned
...
+  "amount": "0.00646600",
+  "taker_side_sell": true
+}
+```
+
+### Buy/Sell, Maker/Taker sides
+For trades, the way exchanges report on the type of trade varies greatly. Some exchanges only report whether a trade was a buy or sell, without specifying which side the trade is actually referring to. Others do specify the side the trade is referring to. With the following mapping, we try to rule out any ambiguous interpretation of the `taker_side_sell` field in our trade endpoints. This mapping is in refererrence to what the exchagnes are reporting via their APIs and in their documentation. We contacted the exchanges for which the documentation did not specify details on the side or for which the meaning was not fully clear. 
+
+#### Maker side 
+The following exchanges report trades from the perspective of the maker being the agressor. In all below cases, it was the maker who was selling.
```
Comment (Legogris) [inline source/index.html.md]: The following exchange report trades from the perspective of maker orders.

File: source/index.html.md [hunk 6]
```diff
@@ -344,6 +344,82 @@ By setting this to `latest`, you will get the most recent version. The returned
...
+| Binance<sup>1</sup>         | `"m": true, false`                         | `"m": true`                                           | 
+| Coinbase Pro<sup>1 2</sup>    | `"side": "buy" or "sell"`                  | `"side": "sell"`                                      |
+| Gemini<sup>2</sup>          | `"type": "buy" or "sell"`                  | `"type": "sell"`                                      | 
+| ZB.com<sup>2</sup>          | `"type": "buy" or "sell"`                  | `"type": "sell"`                                      | 
+<sup>1</sup> Explicitely stated in docs
+
+<sup>2</sup> Confirmed by exchange
+
+#### Taker side
+The following exchanges report trades from the perspective of the taker being the agressor. In all below cases, it was the taker who was selling.
```
Comment (Legogris) [inline source/index.html.md]: See above. Or reword otherwise.

File: source/index.html.md [hunk 7]
```diff
@@ -344,6 +344,82 @@ By setting this to `latest`, you will get the most recent version. The returned
...
+| Bitfinex<sup>2</sup>        | `"type": "sell" or "buy"`                  | `"type":"sell"`                                       | 
+| Ethfinex<sup>2</sup>        | `"type": "sell" or "buy"`                  | `"type":"sell"`                                       | 
+| Huobi<sup>2</sup>           | `"direction": "buy" or "sell"`             | `"direction": "sell"`                                 | 
+| Kraken<sup>2</sup>          | `"b" (buy) "s" (sell)` `"m" (market) "l" (limit)` | `"s"`                                          | Kaiko only reports on buy/sell. Type of order is disregarded. 
+| BitMEX<sup>2</sup>          | `"side": "Sell" or "Buy"`                  | `"side": "Sell"`                                      |
+| Gatecoin<sup>2</sup>        | `way: "bid" or "ask"`                      | `way: "ask"`                                          | 
+| Quoine<sup>1</sup>          | `taker_side: "buy" or "sell"`              | **inverse notation**: `taker_side: "buy"`             | Incorrect notation by Kaiko. Taker was buying. 
+<sup>1</sup> Explicitely stated in docs
```
Comment (Legogris) [inline source/index.html.md]: Explicitely -> explicitly

File: source/index.html.md (hunk 2 above)
Comment (alexandernacho) [reply source/index.html.md]: "This mapping is in reference to what exchanges are reporting via their APIs and documentation, and, in some cases correspondence between Kaiko and the exchange." 

Bit more formal, no?

Comment (Legogris) [reply source/index.html.md]: Works!

**Pull request merged**